tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.PublicApiHasNotChanged.verified.txt

```diff
@@ -256,92 +256,40 @@ namespace Datadog.Trace
     }
     public static class SpanTypes
     {
-        public const string Aerospike = "aerospike";
         public const string Benchmark = "benchmark";
         public const string Build = "build";
         public const string Custom = "custom";
-        public const string GraphQL = "graphql";
         public const string Http = "http";
-        public const string MongoDb = "mongodb";
         public const string Queue = "queue";
-        public const string Redis = "redis";
         public const string Serverless = "serverless";
         public const string Sql = "sql";
         public const string Test = "test";
         public const string Web = "web";
     }
     public static class Tags
     {
-        public const string AmqpCommand = "amqp.command";
-        public const string AmqpDeliveryMode = "amqp.delivery_mode";
-        public const string AmqpExchange = "amqp.exchange";
-        public const string AmqpQueue = "amqp.queue";
-        public const string AmqpRoutingKey = "amqp.routing_key";
         public const string Analytics = "_dd1.sr.eausr";
```

Comment:
Can we also make the following Tags internal?
- Analytics
- Measured
- Origin
- Anything else prefixed with "_dd"

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1968/commits/c5c5d10071de3864efaa62d2a75b7c0c1721d96d